### ISSUE TITLE
Remora parameter question 5hC_5mC vs 5mC

### ISSUE THREAD

**Issue by Eric-CH-Chen** [nanoporetech/megalodon#239]
Dear community,

I am getting back into megalodon and I just a few question with the new models:

1. Is there any accuracy comparison of Remora's 5hC_5mC vs 5mC models? How does it compare with the current Rerio models?
2. With the paradigm shift towards Remora (nanoporetech/remora/issues/2), is it safe to say that the Rerio mod base models (specifically 5mC_cpg, 5mC_all-context, and the 5mC_5hmC_v001) are now outclassed or will soon to be outclassed by the remora models?

Thanks!

**Comment by marcus1487** [comment]
We do not have any direct accuracy comparisons between these two models at the moment, but are working on this. The main issue we have at the moment is an actual ground truth against which to produce such accuracy metrics. The training data for the 5mC+5hmC model (as described in my NCM talk) was produced from 3 samples 1) PCR 2) M.SssI-treatment and 3) M.SssI+TET+NaBH4 treatment. The issue is that the second and third samples are not 100% modified within the CG contexts. Thus these do not provide sufficient accuracy with which to judge accuracy. We are working on synthetic strands to provide more concrete metrics for these models and will report these once they are available. 

These caveats aside, we are confident that the Remora 5mC and 5hmC+5mC CpG models are more accurate than the Rerio (flip-flop modbase) models. Again more details on the methods to validate the new 5hmC model are detailed in my NCM talk. We are aiming to replace the full suite of models in Rerio with Remora models. The one missing at the moment is the all context model. We are working hard to produce a Remora all-context model, but do not have one for release at the moment. Once a Remora all-context model is released these models should be preferred over the Rerio models.

**Comment by Eric-CH-Chen** [comment]
Thank you Marcus for the very helpful reply. For posterity, this is the YouTube link to the video Marcus is referring to (https://www.youtube.com/watch?v=y8nywBuSbpU) so anyone that comes upon this via google can view it.

I am also very interested in follows up to nanoporetech/remora/issues/2 . For context, I am about to do some computationally expensive methylation calls on a large number of samples and I will need to freeze my guppy/megalodon/methylation eventually. Since the video mentions integration in January-2022, I suppose I should wait a little bit longer for the remora models to be validated/published? Any advices welcome!
